README.md

```diff
@@ -27,11 +27,11 @@ This tutorial demonstrates the use of a web client to authenticate to a server u
 - Complete general [example prequisites](prerequisites)
 - Access to a wallet account whose Volta DID Document has a role claim (corresponding to a Switchboard role)
 
-### 1. Run the react client application
-1. Navigate to the React client example: `cd client/react-dapp`
-2. Edit the backend url in `src/config.ts` to `http://localhost:3333`
+### 1. Run the Angular client application
+1. Navigate to the Angular client example: `cd client/angular-dapp`
+2. Edit the backend url in `src/environments/environment.ts` to `http://localhost:3333`
```

Comment:
@dawidgil  I think we can remove this instruction as this seems to be the case by default? What do you think?
```suggestion
```

Reply:
Yeah, you're right. I will remove this step